packable/packable/src/packable/prefix/vec.rs

```diff
@@ -110,12 +110,12 @@ where
     }
 
     #[inline]
-    fn unpack<U: Unpacker, const VERIFY: bool>(
+    fn unpack<U: Unpacker>(
         unpacker: &mut U,
-        visitor: &Self::UnpackVisitor,
+        visitor: Option<&Self::UnpackVisitor>,
     ) -> Result<Self, UnpackError<Self::UnpackError, U::Error>> {
         // The length of any dynamically-sized sequence must be prefixed.
-        let len = B::unpack::<_, VERIFY>(unpacker, &())
+        let len = B::unpack_inner(unpacker, visitor)
```

Comment:
Can you ELI5 me when it's possible to call `unpack` and when not?

Reply:
The same as what?

Reply:
the visitor type of the type you are unpacking